packages/web/modals/ibc-transfer.tsx

```diff
@@ -32,6 +38,22 @@ export const IbcTransferModal: FunctionComponent<ModalBaseProps & IbcTransfer> =
 
     const [didAckWithdrawRisk, setDidAckWithdrawRisk] = useState(false);
 
+    const transferHealth = isWithdraw
+      ? queriesExternalStore.queryIbcStatuses.getIbcStatus(
```

Comment:
Please update this to the new simpler API we made.

Reply:
Is this not the simpler API we made? We pass in the `direction`, `counterPartyChainID`, and `sourceChannelD` in one query?